ports/rp2/main.c

```diff
@@ -181,6 +180,9 @@ int main(int argc, char **argv) {
 
         // Execute user scripts.
         int ret = pyexec_file_if_exists("boot.py");
+
+        tusb_init();
```

Comment:
This needs to be guarded with `#if MICROPY_HW_ENABLE_USBDEV`.

Reply:
Oops, thanks!